examples/Platformio_CO2_MHZ19/README.md

```diff
@@ -0,0 +1,82 @@
+[![PlatformIO](https://github.com/kike-canaries/canairio_sensorlib/workflows/PlatformIO/badge.svg)](https://github.com/kike-canaries/canairio_sensorlib/actions/)
```

Comment:
please remove this readme file. I think that it is better in the main readme file, adding a new lines or new section there.

Reply:
Done